### ISSUE TITLE
Dev: Accessibility: Fix Headers on various pages

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/tdm-calculator#2900]
### Overview
We need to fix the headers so that they have the correct H1, H2, H3, etc. to be WCAG compliant

#### Details
Issues 1: there are two different styling's being applied and we need to know which one is the correct one, so that we can revise the details
<details><summary>Types being utilized</summary>

Type 1, being applied on pages 1, 3, 4, 5 of the calculator
```theme.typography.heading1```
```theme.typography.subHeading```

Type 2, being applied on page 2 of the calculator
```classes.header```
```classes.subtitle```
</details> 

Issue 2: We are not designating all headers and subheaders with the H1 and H2 tags that are necessary for screen readers

<details><summary>P1</summary>

Problems: 
- The primary header is not indicated with an H1 tag, it needs to be added
- The subtitle is indicated with an H3 instead of an H2, even though it is using the theme.typography.subHeading which controls its look.  It needs to be change to H2

P1 DEV URL: https://tdm-dev.azurewebsites.net/calculation/1/0

Inspect on live site

P1 header:
```<div style="font-family: Calibri; font-size: 28px; font-weight: 700; line-height: 40px; letter-spacing: 0.01em; text-align: center; color: rgb(0, 0, 0); margin-top: 0px;"><span>Welcome to Los Angeles’ Transportation Demand Management (TDM) Calculator</span></div>```

P1 subtitle:
```<h3 style="font-family: Calibri; font-size: 20px; font-weight: 400; line-height: 28px; letter-spacing: 0.01em; text-align: center; color: rgb(0, 0, 0);">First, let’s get some information about your project</h3>```

P1 Code URL: https://github.com/hackforla/tdm-calculator/blob/develop/client/src/components/ProjectWizard/WizardPages/ProjectDescriptions.jsx

Code:

```      <div style={theme.typography.heading1}>
        <span>
          Welcome to Los Angeles&rsquo; Transportation Demand Management (TDM)
          Calculator
        </span>
      </div>
      <h3 style={theme.typography.subHeading}>
        First, let&rsquo;s get some information about your project
      </h3>
```

</details> 

<details><summary>P2</summary>

Problems: 
- The primary header is not indicated with an H1 tag, it needs to be added
- The subtitle is indicated with an H3 instead of an H2, even though it is using the theme.typography.subHeading which controls its look.  It needs to be change to H2
- This page uses the old type of classes and needs to be changed
   - Change `classes.header` to `theme.typography.heading1`
   - Change`classes.subtitle` to `theme.typography.subHeading`

P2 DEV URL: https://tdm-dev.azurewebsites.net/calculation/2/0

Inspect on live site

P2 header:
```<div class="header-0-1-648">Determine Project Level</div>```

P2 subtitle:
```<h3 class="subtitle-0-1-651">Project Level (left panel) and Citywide Parking Baseline (next page) are determined by the use specifications entered below.</h3>```

P2 Code URL: https://github.com/hackforla/tdm-calculator/blob/develop/client/src/components/ProjectWizard/WizardPages/ProjectSpecifications.jsx

Code:
```
      <div className={classes.header}>Determine Project Level</div>
      <h3 className={classes.subtitle}>
        Project Level (left panel) and Citywide Parking Baseline (next page) are
        determined by the use specifications entered below.
      </h3>
```
</details>

<details><summary>P3</summary>

Problems: 
- The primary header is not indicated with an H1 tag, it needs to be added
- The subtitle is not indicated with an H2 tag, it needs to be added

P3 URL: https://tdm-dev.azurewebsites.net/calculation/3/0

Inspect on live site

P3 header:
```<div style="font-family: Calibri; font-size: 28px; font-weight: 700; line-height: 40px; letter-spacing: 0.01em; text-align: center; color: rgb(0, 0, 0); margin-top: 0px;"><span>Calculate Project TDM Target Points</span></div>```

P3 subtitle:
```<div style="font-family: Calibri; font-size: 20px; font-weight: 400; line-height: 28px; letter-spacing: 0.01em; text-align: center; color: rgb(0, 0, 0);"><span>Target Points (left panel) may be adjusted based on parking spaces entered below.</span></div>```

P3 Code URL: https://github.com/hackforla/tdm-calculator/blob/develop/client/src/components/ProjectWizard/WizardPages/ProjectTargetPoints.jsx

Code:

```
          <div style={theme.typography.heading1}>
            <span>Calculate Project TDM Target Points</span>
          </div>
          <div style={theme.typography.subHeading}>
            <span>
              Target Points (left panel) may be adjusted based on parking spaces
              entered below.
            </span>
          </div>
```

</details> 

<details><summary>P4</summary>

P4 DEV URL: https://tdm-dev.azurewebsites.net/calculation/4/0

Problems: 
- The subtitle is not indicated with an H2 tag, it needs to be added

Inspect on live site

P4 header:
```<h1 style="font-family: Calibri; font-size: 28px; font-weight: 700; line-height: 40px; letter-spacing: 0.01em; text-align: center; color: rgb(0, 0, 0); margin-top: 0px;"><span>Select TDM Strategies</span></h1>```

P4 subtitle:
```<div style="font-family: Calibri; font-size: 20px; font-weight: 400; line-height: 28px; letter-spacing: 0.01em; text-align: center; color: rgb(0, 0, 0);">Select TDM strategies to earn points to reach the Target (left panel).</div>```

P4 Code URL: https://github.com/hackforla/tdm-calculator/blob/develop/client/src/components/ProjectWizard/WizardPages/ProjectMeasures.jsx

Code:
```
      <h1 style={theme.typography.heading1}>
        <span>Select TDM Strategies</span>
      </h1>
      <div style={theme.typography.subHeading}>
        Select TDM strategies to earn points to reach the Target (left panel).
      </div>
```

</details>

<details><summary>P5 - no problems</summary>

Problems: 
- None

P5 DEV URL: https://tdm-dev.azurewebsites.net/calculation/5/0

Inspect on live site

P5 header:
```<h1 style="font-family: Calibri; font-size: 28px; font-weight: 700; line-height: 40px; letter-spacing: 0.01em; text-align: center; color: rgb(0, 0, 0); margin: 0px; padding: 0px;">TDM Calculation Summary</h1>```

P5 subtitle:
None

P1 Code URL: https://github.com/hackforla/tdm-calculator/blob/develop/client/src/components/ProjectWizard/WizardPages/ProjectSummary/ProjectSummary.jsx

Code:
```
     <h1 style={{ ...theme.typography.heading1, margin: 0, padding: 0 }}>
        TDM Calculation Summary
      </h1>
```
</details>

### Action Items
- [ ] Assign yourself to the development issue.
- [ ] Move the development issue from the Prioritized Backlog to In Progress.
- [ ] [ADD DEVELOPER ACTION ITEMS]
- [ ] Create a PR, if applicable.
- [ ] Create a new slide (or edit one if it already exists) in staging and add the before and after images
- [ ] Once the PR has been approved, use it to close this issue. If there is no PR is associated with this issue, close this issue after lead signoff.

### Resources/Instructions
- 1.01 [ADD LINK TO ACCESSIBILITY PATTERN PAGE FOR HEADINGS]
- 1.02 Add solution here: https://hackforla.github.io/accessibility/known-issues-and-solutions/solutions-index/alert-missing-first-level-heading/

#### Staging and Release decks
- [Staging Deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/)
   - [Staging slide, 2900](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g3cc792692d6_22_7#slide=id.g3cc792692d6_22_7)
- Release Deck
   - [2025-MM-DD Release deck, slide ISSUE NUMBER]

**Comment by Rabia2219** [comment]
Bonnie will update this issue with resource 1.01 when it is ready.

**Comment by ExperimentsInHonesty** [comment]
@entrotech please decide which pattern each header in the calculator should use.  I would prefer if it was a single component, that we could re use.  But if it has to be a seperate one from each page, it should at least use the same styling.  

This issue should both make the styling method consistent across all the calculator pages, and use H1-H6 appropriately.
